great-idea-challenge2/index.html

```diff
@@ -0,0 +1,84 @@
+<!doctype html>
```

Comment:
Great job with flexbox! To get more out of it going forward, I recommend playing around with the `flex-grow` and `flex-basis` properties. These will give you better control than the general `flex` property.